redis/reply.go

```diff
@@ -504,6 +504,26 @@ func Float64Map(result interface{}, err error) (map[string]float64, error) {
 	return m, err
 }
 
+// BytesMap is a helper that converts an array of strings (alternating key, value)
```

Comment:
Not sure what the common use case for this is, the example seems to indicated it would better to use `Values(..)` instead?